### ISSUE TITLE
sample script to add and remove port mirroring session

### ISSUE THREAD

**Issue by aimin-tang** [vmware/pyvmomi-community-samples#445]
Can someone point me to the sample script to configure a port mirroring session? I can manually configure a session and it works. With the pyVmomi script below, I can create a session that appears on vsphere client just like the manual entry, but it does not work! In addition, I cannot find a proper way to remove it.

# creates a session but doesn't seem to work:
```
s_key = '112063'
s_port = vim.dvs.VmwareDistributedVirtualSwitch.VspanPorts(portKey=s_key)
d_key = '112072'
d_port = vim.dvs.VmwareDistributedVirtualSwitch.VspanPorts(portKey=d_key)
configVersion = dvs.config.configVersion
session = vim.dvs.VmwareDistributedVirtualSwitch.VspanSession(name="some-name", 
    enabled=True, stripOriginalVlan=True, normalTrafficAllowed=True, 
    sourcePortTransmitted=s_port, sourcePortReceived=s_port, destinationPort=d_port, 
    sessionType='dvPortMirror', samplingRate=1)
s_spec = vim.dvs.VmwareDistributedVirtualSwitch.VspanConfigSpec(vspanSession=session, 
    operation="add")
c_spec = vim.dvs.VmwareDistributedVirtualSwitch.ConfigSpec(vspanConfigSpec=[s_spec], 
    configVersion=configVersion)
t = dvs.ReconfigureDvs_Task(c_spec)
```

# exception when trying to remove it:
```
configVersion = dvs.config.configVersion
session = vim.dvs.VmwareDistributedVirtualSwitch.VspanSession(name="some-name",)
s_spec = vim.dvs.VmwareDistributedVirtualSwitch.VspanConfigSpec(vspanSession=session, 
    operation="remove")
c_spec = vim.dvs.VmwareDistributedVirtualSwitch.ConfigSpec(vspanConfigSpec=[s_spec], 
    configVersion=configVersion)
t = dvs.ReconfigureDvs_Task(c_spec)
```

I appreciate the help!

**Comment by gyorgypeter** [comment]
Could you solve this issue?

**Comment by Akvinikym** [comment]
To however reads this after me: I think I've managed to solve it - when I did that `edit-save` thing, vSphere set value of parameter `mirroredPacketLength` to `-1`, even though documentation tells that value must not be less than 60. I did the same in my script, and the mirror session works perfectly now.

TL;DR: add `mirroredPacketLength=-1` to the list of mirror session creation arguments
